src/utils/api.js

```diff
@@ -59,6 +59,10 @@ const api = {
     search: q => get("/samples/search/", { q }),
   },
 
+  sampleKinds: {
+    list: () => get("/sample-kinds/"),
```

Comment:
No pagination to handle here? If there is pagination, is the number of sample kinds low & expected to stay the same?

Reply:
In a similar way to ContainerKind, SampleKind is not expected to get much larger. Also, it is not something that will be manageable by interface users. If it happens, I can still add pagination.